### ISSUE TITLE
MissingMethodDocumentation.xq also triggers on tables

### ISSUE THREAD

**Issue by ibruynin** [microsoft/Dynamics365FO-AppChecker#16]
Do we want to execute this rule for "classes" representing tables as well?

```public class <tablename> extends common
{
}```

I'd propose not to do that or at least certainly not for staging tables (since they are generated by VS without comments)
This also seems to overlap with "developerDocumentation" and "Label" properties on the table.

**Comment by quietjay** [comment]
This also triggers on forms despite the form actually having comments in the .xpp. Perhaps it should check /Form/Class/@Comments instead of /Form/@Comments

/// <summary>
/// Some form documentation
/// </summary>
[Form]
public class AppCheckerTestForm extends FormRun
{
    /// <summary>Some more form documentation</summary>
}

**Comment by quietjay** [comment]
> This also triggers on forms despite the form actually having comments in the .xpp. Perhaps it should check /Form/Class/@Comments instead of /Form/@Comments

Actually I think it's triggering on methods __InitMember and __InitDataMethods, perhaps it could filter out methods with GeneratedCodeArtifact="true"
